SPECS/qemu/qemu.spec

```diff
@@ -2099,6 +2093,8 @@ useradd -r -u 107 -g qemu -G kvm -d / -s %{_sbindir}/nologin \
 %{_bindir}/qemu-system-aarch64
 %{_datadir}/systemtap/tapset/qemu-system-aarch64*.stp
 %{_mandir}/man1/qemu-system-aarch64.1*
+%{_datadir}/%{name}/pxe*rom
+%{_datadir}/%{name}/efi*rom
```

Comment:
Double-checking: it seems that now both `qemu-system-aarch64-core` and `qemu-system-x86-core` subpackages will install the same files. Have you verified they don't conflict with each other when installed at the same time?

Also, it might be worth considering moving the common files into a separate subpackage and add `Requires` on it to `qemu-system-aarch64-core` and `qemu-system-x86-core`. That way we are sure to avoid conflicts and remove duplication as a bonus.

Reply:
Thank you for the review and recommendation. Yes, even with the duplication, you are still able to `tdnf install qemu-system-aarch64-core qemu-system-x86-core`. Removing each individually does not remove the romfiles either. However, I do think it is a good idea to remove the duplication anyway. I will create a subpackage named ipxe for this purpose.